README.md

```diff
@@ -36,19 +36,13 @@ for entry in grocy.stock():
 You need a Grocy instance running in demo mode at localhost with https (docker or a php server)
 You can setup url, port and ssl in test/test_const.py
 Easy case : install docker and docker-compose and run :
```

Comment:
I think you need to add an actual new line for the line break to work. So free line before line 38 and line 39.